realtime.go

```diff
@@ -196,8 +196,8 @@ type AlertActivePeriod struct {
 type AlertInformedEntity struct {
 	AgencyID    *string
 	RouteID     *string
-	RouteType   *RouteType
```

Comment:
Can we just add a note that if RouteID is nil, this will be RouteType_Unknown?

Reply:
Good call, added comment!